### PR TITLE
fix(a380x/afs): remove v/s knob push action

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #[issue_no]

## Summary of Changes

The A380X incorrectly carried over the A32NX V/S knob push action. Remove the action.

## References

FCOM:
> When pushed: There is no effect.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Spawn in the air and setup an AP descent or climb. Push the V/S knob and ensure it has no effect. Rotate the V/S knob to set a different V/S and pull it, ensure that V/S is captured by the AP.

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_FCU.xml [hunk 1]
```diff
@@ -291,14 +291,10 @@
                     }
                     (&gt;H:A320_Neo_CDU_VS)
                 </ANTICLOCKWISE_CODE>
-                <PULL_CODE>
+                <LEFT_SINGLE_CODE>
                     (&gt;H:A320_Neo_FCU_VS_PULL)
                     (&gt;H:A320_Neo_CDU_VS)
-                </PULL_CODE>
```
Comment (Saschl) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_FCU.xml:297]: Shouldn't we also remove then the code that handled this? E.g. https://github.com/flybywiresim/aircraft/blob/master/fbw-a380x/src/systems/instruments/src/FCU/Managers/VerticalSpeedManager.ts#L63

Comment (tracernz) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_FCU.xml:297]: Fair point. We might be able to reused some stuff for the ALT button but probably not that.

**Review by Saschl — APPROVED** [review]

**Comment by VeIocityy** [comment]
Trainee Report

Discord Username  : velocity. 
Object of testing : 9174 
Aircraft          : A380X 
Tier of Testing   : 1
Date              : 4/11/2024 

Testing Process: 
1. Loaded in over the Pacific in the air.
2. Turned on the AP @ 300 knots, present heading and climbing.
3. Attempted to put the VS knob in/out with no luck.

Testing Results: 
Not Passed

Negatives:       
No up/down arrows show when hovering around the knob.

**Comment by tracernz** [comment]
Thanks for the test. Brought back the old template for people still using the legacy interaction system.

**Comment by VeIocityy** [comment]
Quality Assurance Trainee Report

Discord Username : velocity.
Object of testing : #9174
Aircraft          : A380X
Tier of Testing   : 1
Date              : 08/11/2024

Testing Process: 
1. Loaded in over the Pacific in the air.
2. Turned on the AP @ 280 knots, present heading and climbing.
3. Tested V/S push. Works as expected.
4. Tested changing VS during open climb, works as expected.

Testing Results: 
Passed

**Pull request merged**